lenovo/thinkpad/z/gen2/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib, pkgs, ... }:
+
+{
+  imports = [
+    ../../../../lenovo/thinkpad/z
+  ];
+
+  boot.initrd.availableKernelModules = [ "nvme" "xhci_pci" "thunderbolt" "usb_storage" "usbhid" "sd_mod" ];
+  boot.initrd.kernelModules = [ "kvm-amd" ];
+
```

Comment:
We are at the moment not duplicating the information from nixos-generate-config:

```suggestion
```

Changing this would be a bit out of scope of this pull request. Also if everything goes to plan, we hopefully see some script that also combines both nixos-generate-config and quirks from nixos-hardware in a single tool.